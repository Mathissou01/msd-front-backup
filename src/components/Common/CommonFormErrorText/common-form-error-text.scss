@use "src/styles/01-settings/settings.colors" as colors;

.c-CommonFormErrorText {
  padding: 2px 0;
  color: colors.$error;
}
