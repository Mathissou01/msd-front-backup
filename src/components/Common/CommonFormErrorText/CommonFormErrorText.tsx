import React from "react";
import "./common-form-error-text.scss";

interface ICommonFormErrorTextProps {
  message: string;
  errorId: string;
}

export default function CommonFormErrorText({
  message,
  errorId,
}: ICommonFormErrorTextProps) {
  return (
    <em className="c-CommonFormErrorText" role="alert" id={errorId}>
      {message}
    </em>
  );
}
