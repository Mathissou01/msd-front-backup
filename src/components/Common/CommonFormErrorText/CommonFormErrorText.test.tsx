import { render } from "@testing-library/react";
import CommonFormErrorText from "./CommonFormErrorText";

const mock = {
  message: "error message",
  errorId: "error-1",
};

describe("CommonFormErrorText", () => {
  it("renders", () => {
    const { container } = render(
      <CommonFormErrorText message={mock.message} errorId={mock.errorId} />,
    );
    expect(container).toMatchSnapshot();
  });
});
