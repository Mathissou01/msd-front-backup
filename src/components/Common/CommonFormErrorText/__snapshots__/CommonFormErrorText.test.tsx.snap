// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommonFormErrorText renders 1`] = `
<div>
  <em
    class="c-CommonFormErrorText"
    id="error-1"
    role="alert"
  >
    error message
  </em>
</div>
`;
